lib/active_model/serializer/adapter/base.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      Base = ActiveModelSerializers::Adapter::Base
```

Comment:
I'm not sure. I'll add the delegator and the warning and if we get an bug out of it, I assume we can revert the changes.